src/lib/babel-config.js

```diff
@@ -4,7 +4,7 @@ export default (env, options={}) => ({
 			loose: true,
 			modules: options.modules || false,
 			uglify: true,
-			browsers: env.browsers ? env.browsers.split() : [
```

Comment:
env.browsers was never passed here, in babel-preset env argument means something different so I've removed it